preconditions/src/main/java/com/palantir/logsafe/exceptions/SafeExceptions.java

```diff
@@ -22,10 +22,10 @@
 /**
  * {@link SafeExceptions} provides utility functionality for SafeLoggable exception implementations.
  */
-final class SafeExceptions {
+public final class SafeExceptions {
     private SafeExceptions() {}
 
-    static String renderMessage(String message, Arg<?>... args) {
+    public static String renderMessage(String message, Arg<?>... args) {
```

Comment:
`message` should be annotated with `@CompileTimeConstant` and renamed to `safeMessage`

Reply:
fixed